dali/operators/operators.cc

```diff
@@ -13,8 +13,18 @@
 // limitations under the License.
 
 #include "dali/core/api_helper.h"
-#include "dali/operators/operators.h"
+#include "dali/operators.h"
 
+/*
+ * The point of these functions is to force the linker to link against dali_operators lib
+ * and not optimize-out symbols from dali_operators
+ */
+
+#ifdef __cplusplus
```

Comment:
Hmm, it's going to be __cplusplus always in a .cc file, unless something goes horribly wrong.

Reply:
xD true that

Reply:
Fixed